src/frontend/package.json

```diff
@@ -50,5 +50,15 @@
     "webpack": "4.42.1",
     "webpack-cli": "3.3.11"
   },
-  "dependencies": {}
+  "dependencies": {
+    "draft-js": "^0.11.5",
```

Comment:
We always pin all the dependencies. We should do that here too IMO

Reply:
Yup. And configure dependabot to test upgrades.

Reply:
I pinned dependencies, I'll configure renovate bot in another PR when this one is merged.